router/Router.js

```diff
@@ -213,13 +213,12 @@ class Router {
         const validator = utapiRequest.getValidator();
         // specific resources will be names of buckets, accounts or users
         const specificResources = validator.get(resourceType);
-
-        const requestContexts = specificResources.map(specificResource =>
+        let requestContexts = [];
```

Comment:
You can still use `const` here as you are not replacing the content of the variable. Another minor change is to rename it to singular `requestContext` since it is only one request context

Reply:
Good catch.